data/json/items/tool/cooking.json

```
@@ -150,8 +150,8 @@
     "symbol": ";",
     "color": "light_gray",
     "ammo": [ "charcoal" ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 200 } } ],
-    "max_charges": 200,
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 375 } } ],
```

It should be rigid, so charcoal won't increase its volume when filled.
Probably the same for the `pseudo_char_smoker` fake tool.
```suggestion
    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "charcoal": 375 } } ],
```

---

I don't think it matters. Once deployed it is a piece of furniture and you can't pick it up with out disassembling it (which turns  it into an item again). When you disassemble it empties itself of food and charcoal automatically and while it is an item you can't load it.

---

Just tested it with `"rigid": true`. I got no errors and the smoker worked, but I can't tell any difference. I can make it rigid though, let me know if you still think it should be.

---

You can leave a smoker as an item and use it. And it grows bigger without a rigid flag.

---

OK, I have made it rigid. Thanks.